### PR TITLE
HOTT-1375: Order customs duties section by sub category

### PR DESCRIPTION
**Pull request by willfish**
### Jira link

https://transformuk.atlassian.net/browse/HOTT-1375

### What?

I have added/removed/altered:

- [x] Added ordering of customs duties
- [x] Removed now-redundant groups of measure types in the measure type file

### Why?

I am doing this because:

- Third country duties were less obvious to users so we wanted to hoist these to
  the top of the customs duties section

### CODE REVIEW

File: app/views/commodities/_commodity.html.erb [hunk 1]
```diff
@@ -23,7 +23,7 @@
           <% end %>
         </div>
         <div class="duty" aria-describedby="commodity-duty-title">
-          <% tcd = commodity.overview_measures.third_country_duty %>
+          <% tcd = commodity.overview_measures.third_country_duties %>
```
Comment (jebw) [inline app/views/commodities/_commodity.html.erb:26]: I think this shifts from using a MeasureCollection to using a Array - not sure that makes any difference though?

**Review by jebw — APPROVED** [review]
Looks good

File: app/views/commodities/_commodity.html.erb (hunk 1 above)
Comment (willfish) [reply app/views/commodities/_commodity.html.erb:26]: I don't think it actually matters for the overview measures. I believe they're also not behind the MeasurePresenter :shrug:

Comment (willfish) [reply app/views/commodities/_commodity.html.erb:26]: Probably better for consistency, though

**Pull request merged**